pallets/dapp-staking/src/test/tests_types.rs

```diff
@@ -2828,6 +2828,7 @@ fn tier_params_check_is_ok() {
             },
         ])
         .unwrap(),
+        slot_number_args: (100, 50),
```

Comment:
Minor suggestion - we can use (1000, 50) to match genesis values.

Reply:
Oh, right, I'll reuse the const.